jans-auth-server/server/src/test/java/io/jans/as/server/BaseComponentTest.java

```diff
@@ -21,4 +42,76 @@ public static void sleepSeconds(int p_seconds) {
             e.printStackTrace();
         }
     }
+
+    public static InumGenerator getInumGenerator() {
+        return (InumGenerator) HM_TEST_INJECTOR.get("InumGenerator");
+    }
+
+    public static StaticConfiguration getStaticConfiguration() {
+        return (StaticConfiguration) HM_TEST_INJECTOR.get("StaticConfiguration");
+    }
+
+    public ClientService getClientService() {
+        return (ClientService) HM_TEST_INJECTOR.get("ClientService");
```

Comment:
Can you check whether `CdiUtil.bean(ClientService.class)` will work instead of alien static holder? If it will work we have to drop this map.